src/goto-symex/symex_decl.cpp

```diff
@@ -90,10 +73,16 @@ void goto_symext::symex_decl(statet &state, const symbol_exprt &expr)
     state.top().hidden_function ||
     state.source.pc->source_location.get_hide();
 
-  target.decl(
```

Comment:
Surprised the target doesn't get told of the declaration anymore?

Reply:
That's a point worth considering, but really `symex_targett::decl` just bloats the formula. I might rather go one step further and remove it completely?!